src/__init__.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+
+from Animals.Bunny import Bunny
+from Animals.Turtle import Turtle
+from Animals.CompetingAnimal import CompetingAnimal
+from Road import Road
+
+havi = CompetingAnimal("Havi the bunny", 0, 3, 'B')
```

Comment:
this is hardcoded, what if I want a competitor with different settings? why should I dive into the heart of the code? read how you can do configuration files in python (like Json) and do that it dynamically creates racers